src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,99 @@
+@use "../../variables" as *;
+.footer {
+  .footer-top {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 52px;
+    .switch {
+      margin-bottom: 20px;
+      .switch-img {
+        width: 150px;
+      }
+    }
+    address {
+      text-align: center;
+      font-size: 14px;
+      line-height: 28px;
```

以下を追加しFigma通りの実装になっているか確認してください
```suggestion
      line-height: 28px;
      margin-bottom: 20px;
```